lib/lte_link_control/lte_lc_modem_hooks.c

```diff
@@ -87,6 +87,12 @@ static void on_modem_init(int err, void *ctx)
 	(void)nrf_modem_at_printf("AT%%FEACONF=0,3,%d",
 		IS_ENABLED(CONFIG_LTE_PLMN_SELECTION_OPTIMIZATION));
 
+	/* Only supported in mfw 2.0.2 and newer.
+	 * Ignore the return value; an error likely means that the feature
+	 * is not supported.
+	 */
+	(void)lte_lc_uiccpowersave_set(CONFIG_LTE_UICCPOWERSAVE_MODE_VALUE);
```

Comment:
It would be easiest to just send the AT command directly. On the other hand, if the configuration is not saved into modem NVM, we could call `lte_lc_uiccpowersave_set()`, but only when the configured mode is something else than default. In that case the error would be appropriate, because it would only be logged when the user tries to enable UICC power saving mode with MFW which does not support it.